qemu/tests/cfg/uefi_check_descriptor_meta-files.cfg

```diff
@@ -0,0 +1,14 @@
+- uefi_check_descriptor_meta-files:
```

Comment:
Can we change the cfg name to something like uefi_pkg.cfg, or some else, in case we might add new cases in future, 
uefi_check_descriptor_meta-files.cfg is especially for meta files check.

Reply:
updated, thanks.